### PR TITLE
Typo

### PR DESCRIPTION
**Pull request by kant**
on #124. Plus minor fixes (proposal)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,11 +25,11 @@ $ go get github.com/garyburd/redigo/redis
...
 $ ./update_deps.sh $buildtype $extra_cmake_flags
 ```
-   $buildtype is optional; it defaults to Release and may be set to any LLVM
+   `$buildtype` is optional; it defaults to Release and may be set to any LLVM
    build type.
-   $extra_cmake_flags is optional. It is passed to CMake.
+   `$extra_cmake_flags` is optional. It is passed to CMake.
 
-2. Run CMake from a build directory:
+2. Run `CMake` from a build directory:
```
Comment (jeroenk) [inline README.md]: CMake is just the name of the tool, and not a command that can be executed, so should not be quoted.

File: README.md [hunk 2]
```diff
@@ -39,23 +39,24 @@ $ cmake -DCMAKE_BUILD_TYPE=$buildtype /path/to/souper
...
 
-3. Run 'make' from the build directory.
-
-4. Optionally run 'make check' to run Souper's test suite. To run the test suite
-   under Valgrind, run 'make check LIT_ARGS="-v --vg --vg-leak"' instead. By
+4. Optionally run `make check` to run Souper's test suite. To run the test suite
+   under Valgrind, run `make check LIT_ARGS="-v --vg --vg-leak"` instead. By
    default the solver is also run under Valgrind. This can be disabled by
-   by adding --vg-arg=--trace-children-skip=/path/to/solver to LIT_ARGS.
+   by adding `--vg-arg=--trace-children-skip=/path/to/solver to LIT_ARGS`.
```
Comment (jeroenk) [inline README.md]: `to' should not be in quotes.

File: README.md [hunk 3]
```diff
@@ -108,15 +109,15 @@ $ /path/to/configure CC=/path/to/sclang CXX=/path/to/sclang++
 $ make
 ```
 
-Compilation using Souper can be sped up by caching queries. By default, Souper
+Compilation using Souper can be speed up by caching queries. By default, Souper
```
Comment (jeroenk) [inline README.md]: `sped` is correct.

**Comment by regehr** [comment]
Closing this one.

**Pull request closed without merging**